amqp/handlers.go

```diff
@@ -61,3 +65,67 @@ func UpdateUserHandler(del amqp.Delivery, dedb, icat *sqlx.DB, configuration *co
 
 	return nil
 }
+
+func UpdateUserBatchHandler(del amqp.Delivery, dedb, icat *sqlx.DB, configuration *config.Config) error {
+	usernames := strings.SplitN(del.RoutingKey[len(BatchUserPrefix)+1:], ".", 2)
```

Comment:
This ensures it's always split into 2. It also means that if we have any usernames with dots in them, hopefully they end up in the second side of a batch if anything. I'm not actually sure it's possible though, so I haven't done anything special to ensure it.